give.php

```diff
@@ -614,6 +614,9 @@ private function includes() {
 			// Include API
 			require_once GIVE_PLUGIN_DIR . 'src/API/API.php';
 
+			// Include Onboarding
+			require_once GIVE_PLUGIN_DIR . 'src/Onboarding/Onboarding.php';
```

Comment:
@kjohnson We stopped loading files like this. every new feature which we add into core will go to the `src` directory which will use the autoload feature and does not need a calling statement.

@JasonTheAdams Did a really awesome job to add e minimal service provider to features. This functionality existing in the https://github.com/impress-org/givewp/tree/epic/paypal-commerce-integration-4886 branch but we can bring over this to `release/2.8.0` for common use.

PR: https://github.com/impress-org/givewp/pull/4895/files

Reply:
While I'd like to get that branch over sooner than later, it's not finished. Please don't merge that branch into anything, yet.

Reply:
I'm happy to refactor this to conform to a new style when it is ready :)